go.sum

```diff
@@ -1,54 +1,83 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
I'd expect that to be the case, but just compiling any of the binaries with this change will cause a large diff in the go.sum.

```
$ g rev-parse HEAD
d2f4a0939309194a7288e20d9e13dcfd1221979b
$ g status
On branch master
Your branch is up to date with 'upstream/master'.

nothing to commit, working tree clean
$ g rev-parse HEAD 
d2f4a0939309194a7288e20d9e13dcfd1221979b
$ go run cmd/keeper/main.go 
2020-01-28T13:04:56.453Z        FATAL   cmd/keeper.go:1894      data dir required
exit status 1
$ g status
On branch master
Your branch is up to date with 'upstream/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   go.sum

no changes added to commit (use "git add" and/or "git commit -a")
$ g diff --stat
 go.sum | 69 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 69 insertions(+)
```

Perhaps a previous committer failed to include their go.sum changes? I guess this requires a separate PR to just fix that issue then.

Reply:
Fwiw I've removed the go.sum change, but now the go.sum in the repo won't reflect what we'll be using in CI (as go will automatically update it). That's probably an issue.

Reply:
@lawrencejones if the go.mod is kept the same then we are just not providing some checksum of the packages but the dependencies will be exactly the same. 

About the different go.sum it could be a missing update by I also saw that the go.sum output is changing between go versions. So we could just update the go.sum generating it using the latest available go version in another PR.